PyPDF2/xmp.py

```diff
@@ -199,11 +202,17 @@ class XmpInformation(PdfObject):
     """
     An object that represents Adobe XMP metadata.
     Usually accessed by :py:attr:`xmp_metadata()<PyPDF2.PdfReader.xmp_metadata>`
+
+    :raises: PdfReadError if XML is invalid
     """
 
     def __init__(self, stream: ContentStream) -> None:
         self.stream = stream
-        doc_root: Document = parseString(self.stream.get_data())
+        try:
+            data = self.stream.get_data()
+            doc_root: Document = parseString(data)
+        except ExpatError:
+            raise PdfReadError("XML in XmpInformation was invalid")
```

Comment:
Consider including diagnostic information contained in the ExpactError object (line, offset, kind of XML issue)/

Example:

```
except ExpatError as e:
    print("Got some XML error. " + str(e))
```